### ISSUE TITLE
Robot URDF needs to be updated to match position of Xtion

### ISSUE THREAD

**Issue by schornakj** [AustinDeric/yak#10]
Using the Ensenso frame works OK for translation. Since the Xtion camera frame isn't in the same position, errors are introduced during rotation which impede accurate reconstruction. There's an established procedure for calibrating sensors mounted on robot arms, right?

**Comment by schornakj** [comment]
Took an initial pass at this by modifying existing variables in my modified godel_robots package.
`<xacro:property name="ensenso_optical_x" value="0.0197"/> 
  <xacro:property name="ensenso_optical_y" value="0.0908"/> 
  <xacro:property name="ensenso_optical_z" value="0.112141"/> `
This helps significantly, though it's still off very slightly since surfaces still very slightly change position when rotating the sensor.

**Comment by schornakj** [comment]
I did an actual calibration of the camera to find its 6-DOF transform relative to robot tool0. I also fixed the interpretation of the camera frame to allow its X and Y axes to match the axes of the camera image, which fixed several other problems throughout the project. There's still work to be done to make sure that the reconstruction is actually _correct_, but this particular issue with the URDF has been resolved.
